docs/source/use/pathways.md

```diff
@@ -90,13 +90,15 @@ This is for advanced users only that know what they're doing - full guidance on
 
 - Get exactly what you want
 - Can be optimized for small image size & fast build times
-- Lots of existing documentation in the SRE world on how to use these
+- Lots of existing documentation in the software research engineer world on how to use these
```

Comment:
I'm not sure this is right. It's also a bit vague, maybe just "Lots of existing documentation on building Dockerfiles"? Alternatively can we link to some relevant "SRE" documentation?

Reply:
SRE also was 'site reliability engineer' here rather than software research engineer.